pom.xml

```diff
@@ -66,8 +66,8 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
     <!-- version for hdds/ozone components -->
     <hdds.version>${ozone.version}</hdds.version>
-    <ozone.version>1.2.0-SNAPSHOT</ozone.version>
-    <ozone.release>Glacier</ozone.release>
+    <ozone.version>1.3.0-SNAPSHOT</ozone.version>
+    <ozone.release>Grand Canyon</ozone.release>
```

Comment:
Is the release for 1.3.0 'Grand Canyon'? I didn't see it on Ozone Roadmap

Reply:
That's the plan, since its the next alphabetical U.S. national park without a confusing name, which seems to be the pattern. But if there's a better one we can use that. I was planning on updating the roadmap after this PR was merged, so approving the PR would function like approving the new name before we use it.

Reply:
Thanks for your explanation